http2_general/test_h2_hpack.py

```diff
@@ -651,26 +649,92 @@ class TestHpackUnknownMethod(TestHpackBase):
         """
     }
 
-    def test_unknown_method_dynamic_table(self):
+    @marks.Parameterize.expand(
+        [marks.Param(name="huffman", huffman=True), marks.Param(name="no_huffman", huffman=False)]
+    )
+    def test_unknown_method_dynamic_table(self, name, huffman):
         """
         Verifies correctness of processing unknown method that
         stored in dynamic table.
         """
         client = self.get_client("deproxy")
         server = self.get_server("deproxy")
-        method = "UPDATEREDIRECTREF"
+        methods = ["UPDATEREDIRECTREF", "PUTA", "GETA", "OPTIONA"]
```

Comment:
You wrote that `tfw_http_meth_str2id` check length of header  in #2362 issue

> In tfw_http_meth_str2id function we check first letter in chunk and the length of header and return method type according to it.

I think we should add a case with `POT` or `GFT` methods. In this case we exclude the error with the length of header.